### DISCUSSION TITLE
Add saving of the selected profile on reboot.

### DISCUSSION THREAD

**Discussion by khseal** [Zer0-bit/gaggiuino#537] [Ideas]
This is a very convenient function so as not to select the desired profile every time.

**Comment by matiasjrossi** [comment]
This feature is already available in latest release/stm32. Long press a profile slot to name and save the current profile. On reboot, the last saved profile will be loaded by default.

**Comment by khseal** [comment]
OK thanks. Sounds like a life hack. =)
